mobile/translations.trm

```diff
@@ -553,4 +553,100 @@
     <value lang="rs" status="initial">Danas</value>
     <value lang="ro" status="initial">Azi</value>
   </text>
+  <text key="january">
+    <context>Calendar circled month January</context>
+    <value lang="en" status="initial">January</value>
+    <value lang="mk" status="initial">Јануари</value>
+    <value lang="de" status="initial">Januar</value>
+    <value lang="rs" status="initial">Januar</value>
+    <value lang="ro" status="translated">Ianuarie</value>
```

Comment:
Why is only RO "translated" ? :D 

we should either mark tham all as initial (when missing a proper translation from a native speaker), or all s "translated" in this case since the translation is easy and we can 100% confirm it is accurate in all languages.